src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball extends Lottery {
```

Comment:
Extending the Ball class from Lottery does not seem appropriate. A ball is not a type of lottery; rather, it's a part of the lottery. Composition would be a better choice here than inheritance.